.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "submodules/stm32-cmake"]
 	path = submodules/stm32-cmake
 	url = https://github.com/ObKo/stm32-cmake.git
+[submodule "submodules/serial"]
+	path = submodules/serial
+	url = https://github.com/wjwwood/serial
```

Comment:
Just my 2c, I would recommend fetching this project at build time using `ExternalProject` or `FetchContent` CMake modules, instead of adding a submodule into esp-serial-flasher.

Reply:
Good idea! This would be a good time to move over the `stm32-cmake` submodule to `FetchContent` as well.

Reply:
Perhaps a find_package script which can do the fetching if needed. In my application the serial library is already present. As a note, I didn't use add_subdirectory on this since it wants catkin as a dependancy, which I don't need here.